_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,302 @@
+```
+
+#### Passed Example 3
+
+The `headers` attribute on the cell refers to `td` with a role of `columnheader` within the same `table`.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<td role="columnheader" id="header1">Projects</td>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td headers="header1">15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 4
+
+A `table` with multiple `columnheader(s)`, where the `headers` attribute on the cell lists the `id(s)` of `th` within the same `table`.
+
+```html
+<table>
+	<tr>
+		<th colspan="2" id="header1">Projects</th>
+		<th colspan="2" id="header2">Exams</th>
+	</tr>
+	<tr>
+		<th id="e1" headers="header1">1</th>
+		<th id="e2" headers="header1">2</th>
+		<th id="p1" headers="header2">1</th>
+		<th id="p2" headers="header2">2</th>
+	</tr>
+	<tr>
+		<td colspan="2" headers="header1 e1 e2">15%</td>
+		<td headers="header2 p1">15%</td>
+		<td headers="header2 p2">45%</td>
+	</tr>
+</table>
+```
+
+#### Passed Example 5
+
+The `headers` attribute on the cell refers to `th` with a role of `rowheader` within the same `table`.
+
+```html
+<table>
+	<tbody>
+		<tr>
+			<th role="rowheader" id="headerAge">Age</th>
+			<td headers="headerAge">65</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 6
+
+The `headers` attribute on the `th` cell refers to other `th` cells within the same `table`. Here the column header has a spans two columns.
+
+```html
+<table>
+	<tr>
+		<th id="name" colspan="2">Name</th>
+	</tr>
+	<tr>
+		<th headers="name">Firsname</th>
+		<th headers="name">Lastname</th>
+	</tr>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `headers` attribute on the cell refers to an element outside the same `table`.
+
+```html
+<span id="elmOutsideTable">Project Costs</span>
+
+<table>
+	<tr>
+		<th id="headerOfColumn">Projects</th>
+	</tr>
+	<tr>
+		<td headers="elmOutsideTable">15%</td>
+	</tr>
+</table>
+```
+
+#### Failed Example 2
+
+One of the cells with `headers` attribute, refers to `th` that does not exist within the same `table`.
+
+```html
+<table>
+	<tr>
+		<th colspan="2" id="header1">Projects</th>
+	</tr>
+	<tr>
+		<th id="e1" headers="header1">1</th>
+		<th id="e2" headers="header1">2</th>
+	</tr>
+	<tr>
+		<td colspan="2" headers="header1 e1 NOT-EXIST">15%</td>
+	</tr>
+</table>
+```
+
+#### Failed Example 3
+
+Cells with `headers` attribute, refers to multiple non-existing elements within the same `table`.
+
+```html
+<table>
+	<tr>
+		<th id="header1">Projects</th>
+		<th id="header2">Exams</th>
+	</tr>
+	<tr>
+		<td headers="NOT-EXIST-1">15%</td>
+		<td headers="NOT-EXIST-2">15%</td>
+	</tr>
+</table>
+```
+
+#### Failed Example 4
+
+The `headers` attribute on the cell refers to an element inside the same `table`, but is not a `rowheader` or `columnheader`.
+
+```html
+<table>
+	<tr>
+		<td>
+			<span id="headerProject">Projects</span>
+		</td>
+	</tr>
+	<tr>
+		<td headers="headerProject">
+			15%
+		</td>
+	</tr>
+</table>
+```
+
+### Inapplicable
```

Comment:
Need an example with a `table` that is not visible (but included in the accessibility tree).